getting-started/setup-multi-node-basic.md

```diff
@@ -23,30 +24,70 @@ node, and follow the instructions in the next section.  More information will be
 forthcoming as TimescaleDB multi-node is made available on these cloud platforms.
 
 ## Initialize data nodes from the access node [](init_data_nodes_on_access_node)
-While connected to the access node (psql), use the command:
+
+Once logged in on the access node, it is necessary to add data nodes
+to the local database before creating distributed hypertables. This
+will make the data node available for use by distributed hypertables
+and the access node will also connect to the data node and initialize
+it.
+
+While connected to the access node as a superuser (e.g., via `psql`),
+use the command:
 
 ```sql
 SELECT add_data_node('example_node_name', host => 'example_host_address');
 ```
 
-`example_node_name` should be a unique name for the node. `example_host_address` 
-is the host name or IP address of the data node.
+`example_node_name` should be a unique name for the
+node. `example_host_address` is the host name or IP address of the
+data node. It is also possible to specify a password to authenticate
+with using the optional `password` parameter. But this is only
+necessary if password authentication is used to connect to data nodes
+and the password is not provided through other means (e.g., a local
+password file). See the [`add_data_node`][add_data_node] API reference
+documentation for detailed information about this command.
+
+It should now be possible to create distributed hypertables. Note,
+however, that in order to create and use distributed hypertables as a
+non-superuser, the user role needs to be distributed as well (i.e.,
+exist on all data nodes) and requires privileges to use the data nodes
+on the access node. Please refer to the next section for instruction
+on how to create distributed roles and grant data node privileges.
 
-The system should be up and running at this point.
+## (Optional) Add distributed user roles
 
-## (Optional) Add user roles to the distributed database
+Like any other database object, roles are _not_ distributed by
+default. In other words, a newly created role on the access node will
```

Comment:
Hypertables and chunks are distributed, so this is not entirely true. Suggest to reformulate. (Just discussing roles is fine.)

Reply:
No, Hypertables and chunks are not distributed by default. You have to explicitly create a distributed hypertable, as opposed to a regular hypertable, for them to be distributed.

Reply:
I think you should just remove the first sentence, it is redundant.